app/assets/stylesheets/active_admin/components/_status_tags.scss

```diff
@@ -6,10 +6,6 @@
   padding: 3px 5px 2px 5px;
   font-size: 0.8em;
 
-  &.ok, &.published, &.complete, &.completed, &.green { background: #8daa92; }
-  &.warn, &.warning, &.orange { background: #e29b20; }
-  &.error, &.errored, &.red { background: #d45f53; }
-
```

Comment:
I don't think it makes sense to keep compatibility with a deprecation, but remove the styles, since people's designs would break anyways. I think we should add TODO note here to "schedule removal" and we actually remove the feature. Would that get in the middle somehow of the rest of the css overhaul?